storage/sparse_merkle/src/lib.rs

```diff
@@ -21,7 +21,7 @@
 
 #[cfg(test)]
 mod mock_tree_store;
-mod nibble_path;
+pub mod nibble_path;
```

Comment:
Why?

Reply:
Jellyfish crate should be able to use it, temporarily though.

Reply:
This change can go into the commit that starts using nibble path in the new library.

Reply:
@wqfish It is already used. What do you mean?